app/controllers/policies_controller.rb

```diff
@@ -6,7 +6,7 @@ class PoliciesController < ApplicationController
   def index; end
 
   def new
-    policy.programme = true if params[:programme]
+    policy.sub_policy = true if params[:programme]
```

Comment:
Hasn't the parameter name changed from "programme" to "sub_policy", too?  (so `params[:programme]` should become `params[:sub_policy]`)